safecast_deploy/same_env.py

```diff
@@ -46,22 +47,26 @@ def _generate_result(self):
                 'old_version': self.state.env_metadata[self.state.subenvs['web']]['version'],
                 'old_version_parsed': self.state.old_versions_parsed['web'],
             },
-            'wrk': {
+        }
+        self._add_git('web', result)
+
+        if self.state.has_worker:
+            result['wrk'] = {
                 'env': self.state.env_metadata[self.state.subenvs['wrk']]['name'],
                 'new_version': self.state.new_version,
                 'new_version_parsed': self.state.new_versions_parsed['wrk'],
                 'old_version': self.state.env_metadata[self.state.subenvs['wrk']]['version'],
                 'old_version_parsed': self.state.old_versions_parsed['wrk'],
-            },
-        }
-        self._add_git('web', result)
-        self._add_git('wrk', result)
+            }
+            self._add_git('wrk', result)
+
         return result
 
     def _add_git(self, role, result):
         repo_names = {
             'api': 'safecastapi',
             'ingest': 'ingest',
+            'reporting': 'reporting2',
```

Comment:
Just checking, in the `new_env` changes `reporting` is used as the repo name, here it's `reporting2`